### ISSUE TITLE
504 gateway time-out after reboot

### ISSUE THREAD

**Issue by cyber-mac** [andreapollastri/cipi#190]
Deployed a VM (Google Cloud - f1 instance w/ 614MB ram) with Ubuntu 20.04 LTS, all packages were up-to-date.  Installed CIPI, the Web UI was accessible and working post install.  While still connected via SSH after installing CIPI I initiated a system reboot.  After which the VM becomes inaccessible, unable to SSH, the CIPI Web UI is not accessible -- when accessing the site via IP address all that is returned is 504 - Gateway Time-out.

Even the local console is not accessible...   I was able to capture the following error message from the console:
[ 2170.372798] Out of memory: Killed process 1154 (php) total-vm:120284kB, anon-rss:19156kB, file-rss:3592kB, shmem-rss:0kB, UID:1002 pgtables:220kB oom_score_adj:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372482] supervisord invoked oom-killer: gfp_mask=0x100cca(GFP_HIGHUSER_MOVABLE), order=0, oom_score_adj=0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372487] CPU: 0 PID: 558 Comm: supervisord Not tainted 5.4.0-1040-gcp #43-Ubuntu
Apr  6 22:28:06 myserver1 kernel: [ 2170.372488] Hardware name: Google Google Compute Engine/Google Compute Engine, BIOS Google 01/01/2011
Apr  6 22:28:06 myserver1 kernel: [ 2170.372489] Call Trace:
Apr  6 22:28:06 myserver1 kernel: [ 2170.372498]  dump_stack+0x6d/0x8b
Apr  6 22:28:06 myserver1 kernel: [ 2170.372501]  dump_header+0x4f/0x1eb
Apr  6 22:28:06 myserver1 kernel: [ 2170.372502]  oom_kill_process.cold+0xb/0x10
Apr  6 22:28:06 myserver1 kernel: [ 2170.372506]  out_of_memory.part.0+0x1df/0x3d0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372507]  out_of_memory+0x6d/0xd0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372509]  __alloc_pages_slowpath+0xd5e/0xe50
Apr  6 22:28:06 myserver1 kernel: [ 2170.372513]  ? blk_flush_plug_list+0xe3/0x110
Apr  6 22:28:06 myserver1 kernel: [ 2170.372515]  __alloc_pages_nodemask+0x2d0/0x320
Apr  6 22:28:06 myserver1 kernel: [ 2170.372518]  alloc_pages_current+0x87/0xe0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372519]  __page_cache_alloc+0x72/0x90
Apr  6 22:28:06 myserver1 kernel: [ 2170.372520]  pagecache_get_page+0xbf/0x300
Apr  6 22:28:06 myserver1 kernel: [ 2170.372521]  filemap_fault+0x6b2/0xa50
Apr  6 22:28:06 myserver1 kernel: [ 2170.372524]  ? unlock_page_memcg+0x12/0x20
Apr  6 22:28:06 myserver1 kernel: [ 2170.372527]  ? page_add_file_rmap+0xff/0x1a0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372528]  ? xas_load+0xd/0x80
Apr  6 22:28:06 myserver1 kernel: [ 2170.372529]  ? xas_find+0x17f/0x1c0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372530]  ? filemap_map_pages+0x24c/0x380
Apr  6 22:28:06 myserver1 kernel: [ 2170.372534]  ext4_filemap_fault+0x32/0x50
Apr  6 22:28:06 myserver1 kernel: [ 2170.372537]  __do_fault+0x3c/0x130
Apr  6 22:28:06 myserver1 kernel: [ 2170.372538]  do_fault+0x24b/0x640
Apr  6 22:28:06 myserver1 kernel: [ 2170.372541]  ? __switch_to_asm+0x34/0x70
Apr  6 22:28:06 myserver1 kernel: [ 2170.372543]  __handle_mm_fault+0x4c5/0x7a0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372545]  handle_mm_fault+0xca/0x200
Apr  6 22:28:06 myserver1 kernel: [ 2170.372547]  do_user_addr_fault+0x1f9/0x450
Apr  6 22:28:06 myserver1 kernel: [ 2170.372549]  __do_page_fault+0x58/0x90
Apr  6 22:28:06 myserver1 kernel: [ 2170.372550]  do_page_fault+0x2c/0xe0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372551]  page_fault+0x34/0x40
Apr  6 22:28:06 myserver1 kernel: [ 2170.372554] RIP: 0033:0x569ef6
Apr  6 22:28:06 myserver1 kernel: [ 2170.372560] Code: Bad RIP value.
Apr  6 22:28:06 myserver1 kernel: [ 2170.372561] RSP: 002b:00007ffca17ef0a0 EFLAGS: 00010206
Apr  6 22:28:06 myserver1 kernel: [ 2170.372562] RAX: 000000000000026b RBX: 00007f82398df400 RCX: 0000000000000312
Apr  6 22:28:06 myserver1 kernel: [ 2170.372563] RDX: 0000000000000000 RSI: 000000000000006b RDI: 0000000000000002
Apr  6 22:28:06 myserver1 kernel: [ 2170.372564] RBP: 00007f82398df5b0 R08: 000000000000006b R09: 0000000000002ca0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372565] R10: 0000000000000130 R11: 0000000000000000 R12: 0000000001e79750
Apr  6 22:28:06 myserver1 kernel: [ 2170.372565] R13: 0000000000000000 R14: 0000000000000002 R15: 00007f8239914724
Apr  6 22:28:06 myserver1 kernel: [ 2170.372566] Mem-Info:
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574] active_anon:103993 inactive_anon:1442 isolated_anon:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574]  active_file:288 inactive_file:740 isolated_file:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574]  unevictable:4635 dirty:0 writeback:0 unstable:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574]  slab_reclaimable:5647 slab_unreclaimable:12790
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574]  mapped:4236 shmem:4623 pagetables:2529 bounce:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372574]  free:7768 free_pcp:30 free_cma:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372577] Node 0 active_anon:415972kB inactive_anon:5768kB active_file:1152kB inactive_file:2960kB unevictable:18540kB isolated(anon):0kB isolated(file):0kB mapped:16944kB dirty:0kB writeback:0kB shmem:18492kB shmem_thp: 0kB shmem_pmdmapped: 0kB anon_thp: 0kB writeback_tmp:0kB unstable:0kB all_unreclaimable? no
Apr  6 22:28:06 myserver1 kernel: [ 2170.372577] Node 0 DMA free:2988kB min:784kB low:980kB high:1176kB active_anon:12036kB inactive_anon:0kB active_file:0kB inactive_file:52kB unevictable:0kB writepending:0kB present:15920kB managed:15836kB mlocked:0kB kernel_stack:44kB pagetables:168kB bounce:0kB free_pcp:0kB local_pcp:0kB free_cma:0kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372580] lowmem_reserve[]: 0 553 553 553 553
Apr  6 22:28:06 myserver1 kernel: [ 2170.372581] Node 0 DMA32 free:28084kB min:28100kB low:35124kB high:42148kB active_anon:403936kB inactive_anon:5768kB active_file:1152kB inactive_file:2908kB unevictable:18540kB writepending:0kB present:611532kB managed:574456kB mlocked:18540kB kernel_stack:4116kB pagetables:9948kB bounce:0kB free_pcp:120kB local_pcp:120kB free_cma:0kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372584] lowmem_reserve[]: 0 0 0 0 0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372585] Node 0 DMA: 9*4kB (E) 7*8kB (UME) 11*16kB (ME) 3*32kB (ME) 3*64kB (UME) 5*128kB (UE) 1*256kB (E) 1*512kB (E) 1*1024kB (U) 0*2048kB 0*4096kB = 2988kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372590] Node 0 DMA32: 5*4kB (E) 126*8kB (UME) 303*16kB (UME) 240*32kB (UME) 131*64kB (UME) 46*128kB (UME) 1*256kB (M) 0*512kB 0*1024kB 0*2048kB 0*4096kB = 28084kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372596] Node 0 hugepages_total=0 hugepages_free=0 hugepages_surp=0 hugepages_size=1048576kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372597] Node 0 hugepages_total=0 hugepages_free=0 hugepages_surp=0 hugepages_size=2048kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372597] 7708 total pagecache pages
Apr  6 22:28:06 myserver1 kernel: [ 2170.372598] 0 pages in swap cache
Apr  6 22:28:06 myserver1 kernel: [ 2170.372599] Swap cache stats: add 0, delete 0, find 0/0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372599] Free swap  = 0kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372600] Total swap = 0kB
Apr  6 22:28:06 myserver1 kernel: [ 2170.372600] 156863 pages RAM
Apr  6 22:28:06 myserver1 kernel: [ 2170.372601] 0 pages HighMem/MovableOnly
Apr  6 22:28:06 myserver1 kernel: [ 2170.372601] 9290 pages reserved
Apr  6 22:28:06 myserver1 kernel: [ 2170.372601] 0 pages cma reserved
Apr  6 22:28:06 myserver1 kernel: [ 2170.372602] 0 pages hwpoisoned
Apr  6 22:28:06 myserver1 kernel: [ 2170.372602] Tasks state (memory values in pages):
Apr  6 22:28:06 myserver1 kernel: [ 2170.372602] [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
Apr  6 22:28:06 myserver1 kernel: [ 2170.372608] [    161]     0   161    12921      681   102400        0          -250 systemd-journal
Apr  6 22:28:06 myserver1 kernel: [ 2170.372610] [    178]     0   178      622      145    45056        0             0 none
Apr  6 22:28:06 myserver1 kernel: [ 2170.372611] [    242]     0   242     4732      806    65536        0         -1000 systemd-udevd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372614] [    367]     0   367    70053     4501    90112        0         -1000 multipathd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372615] [    475]   100   475     6701      866    77824        0             0 systemd-network
Apr  6 22:28:06 myserver1 kernel: [ 2170.372617] [    479]   101   479     6026     1715    86016        0             0 systemd-resolve
Apr  6 22:28:06 myserver1 kernel: [ 2170.372618] [    523]     0   523    60264      483   102400        0             0 accounts-daemon
Apr  6 22:28:06 myserver1 kernel: [ 2170.372619] [    532]   103   532     1869      717    57344        0          -900 dbus-daemon
Apr  6 22:28:06 myserver1 kernel: [ 2170.372620] [    537]     0   537   178316     1316   176128        0             0 google_osconfig
Apr  6 22:28:06 myserver1 kernel: [ 2170.372622] [    542]     0   542     7323     2757    94208        0             0 networkd-dispat
Apr  6 22:28:06 myserver1 kernel: [ 2170.372623] [    546]     0   546    60105     2955   212992        0             0 php-fpm7.3
Apr  6 22:28:06 myserver1 kernel: [ 2170.372625] [    547]     0   547    60172     3028   204800        0             0 php-fpm7.4
Apr  6 22:28:06 myserver1 kernel: [ 2170.372626] [    548]     0   548    59921     3040   217088        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372628] [    550]   104   550    56125      731    86016        0             0 rsyslogd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372629] [    557]     0   557   159850     3103   217088        0          -900 snapd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372630] [    558]     0   558     7899     4131    90112        0             0 supervisord
Apr  6 22:28:06 myserver1 kernel: [ 2170.372632] [    562]     0   562      951      518    49152        0             0 atd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372633] [    563]     0   563    99090     2988   139264        0             0 f2b/server
Apr  6 22:28:06 myserver1 kernel: [ 2170.372634] [    580]   112   580     3303      475    49152        0             0 chronyd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372636] [    587]     0   587   178772     1135   172032        0          -999 google_guest_ag
Apr  6 22:28:06 myserver1 kernel: [ 2170.372638] [    601]   112   601     1221       72    49152        0             0 chronyd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372639] [    602]     0   602     1457      358    49152        0             0 agetty
Apr  6 22:28:06 myserver1 kernel: [ 2170.372640] [    636]     0   636    14329      375   106496        0             0 nginx
Apr  6 22:28:06 myserver1 kernel: [ 2170.372641] [    637]    33   637    14553      875   110592        0             0 nginx
Apr  6 22:28:06 myserver1 kernel: [ 2170.372643] [    644]   114   644    12925      500    90112        0             0 redis-server
Apr  6 22:28:06 myserver1 kernel: [ 2170.372644] [    662]    33   662    60206     1843   192512        0             0 php-fpm7.3
Apr  6 22:28:06 myserver1 kernel: [ 2170.372645] [    663]    33   663    60206     1843   192512        0             0 php-fpm7.3
Apr  6 22:28:06 myserver1 kernel: [ 2170.372646] [    666]    33   666    60273     1898   192512        0             0 php-fpm7.4
Apr  6 22:28:06 myserver1 kernel: [ 2170.372648] [    667]    33   667    60273     1898   192512        0             0 php-fpm7.4
Apr  6 22:28:06 myserver1 kernel: [ 2170.372649] [    675]     0   675    59349      471    94208        0             0 polkitd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372650] [    702]     0   702    27024     2575   106496        0             0 unattended-upgr
Apr  6 22:28:06 myserver1 kernel: [ 2170.372651] [    752]     0   752     3047      675    57344        0         -1000 sshd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372652] [    776]     0   776     4224      909    77824        0             0 systemd-logind
Apr  6 22:28:06 myserver1 kernel: [ 2170.372654] [    793]     0   793     2137      505    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372655] [   1120]     0  1120      652      106    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372656] [   1121]     0  1121      652      104    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372657] [   1123]     0  1123    29559     5404   217088        0             0 php8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372659] [   1124]     0  1124    29559     5352   221184        0             0 php8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372660] [   1125]  1002  1125    30071     5525   225280        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372661] [   1126]  1002  1126    30071     5511   221184        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372662] [   1127]  1002  1127    29559     5432   217088        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372663] [   1128]  1002  1128    30071     5339   212992        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372664] [   1129]  1002  1129    29559     5303   212992        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372665] [   1130]  1002  1130    30071     5448   225280        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372666] [   1131]     0  1131     2558      622    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372667] [   1143]     0  1143      652      122    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372668] [   1144]     0  1144    29559     5374   217088        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372669] [   1152]     0  1152     3453     1038    65536        0             0 sshd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372670] [   1154]  1002  1154    30071     5687   225280        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372671] [   1155]  1002  1155    30071     5425   225280        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372673] [   1157]  1001  1157     4654     1120    73728        0             0 systemd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372674] [   1158]  1001  1158    42329      918    90112        0             0 (sd-pam)
Apr  6 22:28:06 myserver1 kernel: [ 2170.372675] [   1164]     0  1164      652      107    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372676] [   1165]     0  1165      624      372    45056        0             0 run-parts
Apr  6 22:28:06 myserver1 kernel: [ 2170.372678] [   1191]     0  1191     2174      498    53248        0             0 mysql-systemd-s
Apr  6 22:28:06 myserver1 kernel: [ 2170.372679] [   1197]     0  1197      652      410    45056        0             0 update-motd-hwe
Apr  6 22:28:06 myserver1 kernel: [ 2170.372680] [   1205]     0  1205     2174       77    53248        0             0 mysql-systemd-s
Apr  6 22:28:06 myserver1 kernel: [ 2170.372682] [   1206]   113  1206    25025     3343   176128        0             0 mysqld
Apr  6 22:28:06 myserver1 kernel: [ 2170.372683] [   1210]     0  1210     3005      804    61440        0             0 apt-config
Apr  6 22:28:06 myserver1 kernel: [ 2170.372684] [   1213]     0  1213     2558      545    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372685] [   1214]     0  1214     2558      545    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372686] [   1217]     0  1217     2558      545    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372687] [   1218]     0  1218      652      106    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372688] [   1219]     0  1219      652      105    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372689] [   1220]     0  1220      652      107    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372691] [   1221]     0  1221    16323     1100   118784        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372692] [   1222]     0  1222    16323      987   114688        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372693] [   1223]     0  1223    16323     1040   118784        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372694] [   1224]     0  1224     2558      427    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372695] [   1225]     0  1225      652      105    49152        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372696] [   1226]     0  1226    16323     1013   114688        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372697] [   1227]     0  1227     2558      476    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372699] [   1228]     0  1228     2558      591    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372700] [   1229]     0  1229      652       90    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372701] [   1230]     0  1230    16323     1041   118784        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372702] [   1231]     0  1231     2558      496    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372703] [   1232]     0  1232      652       90    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372704] [   1233]     0  1233    16323     1056   110592        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372705] [   1234]     0  1234     2558      536    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372706] [   1237]     0  1237     2558      533    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372708] [   1238]     0  1238     2558      533    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372709] [   1239]     0  1239     2558      533    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372710] [   1240]     0  1240      652      124    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372711] [   1241]     0  1241    16323     1019   122880        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372712] [   1242]     0  1242      652      103    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372713] [   1243]     0  1243    16323     1164   122880        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372714] [   1244]     0  1244      652      106    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372715] [   1245]     0  1245      652      102    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372716] [   1246]     0  1246      652      106    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372717] [   1247]     0  1247    16323     1125   114688        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372718] [   1248]     0  1248    16323     1129   110592        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372719] [   1249]     0  1249    16323     1072   122880        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372720] [   1250]     0  1250     2558      507    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372721] [   1252]     0  1252     3047      714    65536        0             0 sshd
Apr  6 22:28:06 myserver1 kernel: [ 2170.372722] [   1253]     0  1253      652       90    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372723] [   1254]     0  1254    16323     1118   114688        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372725] [   1256]     0  1256     2558      443    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372725] [   1257]     0  1257     2558      471    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372727] [   1258]    33  1258    59591     2154   212992        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372728] [   1259]    33  1259    60021     2187   196608        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372729] [   1260]    33  1260    60021     2188   196608        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372730] [   1261]     0  1261     2558      471    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372731] [   1265]     0  1265      652      106    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372732] [   1266]     0  1266     2558      627    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372733] [   1267]     0  1267    11728      415    65536        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372734] [   1268]     0  1268      652      103    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372736] [   1269]     0  1269      652      104    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372737] [   1270]     0  1270    11728      544    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372738] [   1271]     0  1271    11728      354    69632        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372739] [   1272]     0  1272      652      107    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372740] [   1273]     0  1273    11728      307    81920        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372741] [   1274]     0  1274     2558      487    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372742] [   1278]     0  1278      652      106    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372743] [   1279]     0  1279    11728      348    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372744] [   1280]     0  1280     2558      540    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372745] [   1281]     0  1281     2558      527    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372746] [   1282]     0  1282      652       88    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372747] [   1283]     0  1283    11728      404    81920        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372748] [   1284]     0  1284     2558      480    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372749] [   1287]     0  1287     2558      550    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372750] [   1288]     0  1288      652       67    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372751] [   1289]     0  1289    11728      293    81920        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372752] [   1290]     0  1290     2558      547    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372753] [   1291]    33  1291    60021     2109   196608        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372755] [   1292]     0  1292     2558      547    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372756] [   1293]     0  1293      652      106    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372757] [   1294]     0  1294      652      102    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372758] [   1296]     0  1296    11728      371    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372759] [   1297]     0  1297    11728      469    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372760] [   1299]     0  1299     2558      605    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372761] [   1301]     0  1301     2558      484    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372762] [   1302]    33  1302    60021     2102   196608        0             0 php-fpm8.0
Apr  6 22:28:06 myserver1 kernel: [ 2170.372763] [   1303]     0  1303      652      102    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372764] [   1304]     0  1304      652      104    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372765] [   1305]     0  1305    11728      293    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372766] [   1306]     0  1306    11728      318    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372767] [   1307]     0  1307     2558      646    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372768] [   1309]     0  1309     2558      588    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372769] [   1310]     0  1310     2558      507    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372770] [   1311]     0  1311      652       90    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372771] [   1312]     0  1312      652      102    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372772] [   1313]     0  1313    11728      549    81920        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372774] [   1314]     0  1314    11728      448    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372775] [   1315]     0  1315      652      106    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372776] [   1316]     0  1316    11728       71    69632        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372777] [   1317]     0  1317      652      106    40960        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372778] [   1318]     0  1318    11728       70    69632        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372779] [   1320]     0  1320      652      105    45056        0             0 sh
Apr  6 22:28:06 myserver1 kernel: [ 2170.372780] [   1321]     0  1321    11728      211    73728        0             0 php
Apr  6 22:28:06 myserver1 kernel: [ 2170.372781] [   1322]     0  1322     2426      581    53248        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372782] [   1323]     0  1323     2295      458    49152        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372783] [   1324]     0  1324     2270      423    49152        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372784] [   1326]     0  1326     2137       68    49152        0             0 cron
Apr  6 22:28:06 myserver1 kernel: [ 2170.372785] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/system.slice/supervisor.service,task=php,pid=1154,uid=1002
Apr  6 22:28:06 myserver1 kernel: [ 2170.372798] Out of memory: Killed process 1154 (php) total-vm:120284kB, anon-rss:19156kB, file-rss:3592kB, shmem-rss:0kB, UID:1002 pgtables:220kB oom_score_adj:0
Apr  6 22:28:06 myserver1 kernel: [ 2170.389407] oom_reaper: reaped process 1154 (php), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB

**Comment by cyber-mac** [comment]
Further to -- this is repeatable, as I destroyed the VM, created a new VM, ensured all packages were updated -- did a system reboot and the VM was still accessible.  Installed CIPI and following installation everything was functional.  Perform a system reboot and as previously mentioned the VM becomes inaccessible, unable to SSH, the CIPI Web UI returned 504 Gateway time-out.  VM console access is also useless, the VM is pretty much garbage at this point.
